gemdaq-testing/gemreadout/src/common/GEMDataParker.cc

```diff
@@ -151,44 +146,48 @@ uint32_t* gem::readout::GEMDataParker::getGLIBData(
       if (contqueue%kUPDATE7 == 0 &&  contqueue != 0) {
         contvfats_++;
         /*
-        INFO(" ::getGLIBData conter " << contqueue << " contvfats " << contvfats_ << " dataque.size " << dataque.size() 
+        INFO(" ::getGLIBData conter "  << contqueue << " contvfats " << contvfats_ << " dataque.size " << dataque.size() 
         */
       }
     }
 
-    uint32_t* pDQ = gem::readout::GEMDataParker::GEMEventMaker(bufferCount);
-    Counter[0] = *(pDQ+0);
-    Counter[2] = event_; // *(pDQ+2);
-    Counter[3] = *(pDQ+3);
+    uint32_t* pDQ = gem::readout::GEMDataParker::GEMEventMaker(Counter);
```

Comment:
This is still happening inside the "get data from GLIB" loop, which means, that the loop won't continue until this function has returned.
This is still a potentially massive bottleneck.  I thought we discussed how to improve this splitting